core/domain/feedback_domain.py

```diff
@@ -97,25 +161,54 @@ def to_dict(self):
 class FeedbackAnalytics(object):
     """Domain object representing feedback analytics
     for a specific exploration.
+    
+    Attributes:
+        exploration_id: str. The associated exploration's ID.
+        num_open_threads: int. The number of open threads under 
+            a given exploration.
+        num_total_threads: int. The number of total threads under 
+            a given exploration (regardless of status).
     """
 
     def __init__(self, exploration_id, num_open_threads, num_total_threads):
+        """Initializes a FeedbackAnalytics object.
+        """
         self.id = exploration_id
         self.num_open_threads = num_open_threads
         self.num_total_threads = num_total_threads
 
     def to_dict(self):
+        """
+        
+        Returns:
+            list(dict). Dicts mapping key-value pairs based on the 
+            corresponding data fetched regarding the number of open and total
+            threads respectively.
+        """
         return {
             'num_open_threads': self.num_open_threads,
             'num_total_threads': self.num_total_threads
         }
 
 
 class Suggestion(object):
-    """Domain object for a suggestion."""
+    """Domain object for a suggestion.
+    
+    Attributes:
+        full_thread_id: str. The suggestion thread ID.
+        author_id: str. The ID of the message's author.
+        exploration_id: str. The associated exploration's ID.
+        exploration_version: This represents the version of the exploration 
+            associated with the suggestion.
+        state_name: The name of the state attached to the suggestion.
+        description: A description of the change suggestion.
+        state_content: The state's "suggested" content.
+    """
 
     def __init__(self, full_thread_id, author_id, exploration_id,
                  exploration_version, state_name, description, state_content):
+        """Initializes a Suggestion object. 
+        """
```

Comment:
This was handled by attributes and left alone per feedback from @seanlip on Dec 24, 2016. So I'll say this is done.